test_util/cluster.py

```diff
@@ -256,9 +252,6 @@ def install_dcos(
     install_prereqs=True,
     install_prereqs_only=False,
 ):
-    if cluster.provider != 'onprem':
-        raise NotImplementedError('Install is only supported for onprem clusters')
```

Comment:
There is no such thing as an onprem install from cloudformation, so this message will never get hit. Documentation should be improved to start that cloudformation is really "DCOS provided via cloudformation template" as opposed to the incorrect interpretation of DC/OS installed on-prem using a cloudformation VPC (which is the from_vpc method)